test/CMakeLists.txt

```diff
@@ -11,7 +11,10 @@ function(add_test_simple_schema name schema instance)
         WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
 endfunction()
 
-file(GLOB TEST_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/*)
+add_subdirectory(JSON-Schema-Test-Suite)
+add_subdirectory(cmake-install)
+
+file(GLOB TEST_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/issue-*)
```

Comment:
Yes, it relates to the mentioned command [conan create](https://docs.conan.io/en/1.52/reference/commands/creator/create.html). Because this executes cmake inside the source folder, i.e. not in a specific `build` sub-directory. And so the existing glob command failed because it added temporary cmake directories also.

Maybe I can configure conan to perform CMake out of source builds when executing `conan create .` by adding a package layout config (https://docs.conan.io/en/latest/reference/conanfile/tools/cmake/cmake_layout.html#cmake-layout). 

Please wait for an update on this pull request.

Reply:
Adding a `layout` method to the `conanfile.py` makes the change on `test/CMakeLists.txt` no longer required.